GraphQL-README.md

```diff
@@ -76,7 +76,7 @@ query BrandsQuery(
 }
 ```
 
-## individual bank
+## individual bank by tag
```

Comment:
Also need to update the line below from
`brand(tag: $tag) {`
to 
`brandByTag(tag: $tag)`